.env.example

```
@@ -0,0 +1,14 @@
+# LLM Configuration
+# Model name for development (e.g., gpt-3.5-turbo, gpt-4, claude-3-sonnet-20240229, gemini-pro)
+DEV_MODEL_NAME=gpt-3.5-turbo
+DEV_LLM_NAME=gpt-3.5-turbo
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The DEV_LLM_NAME key should go before the DEV_MODEL_NAME key

(UnorderedKey)